src/scene.ts

```diff
@@ -4139,7 +4138,6 @@ export class Scene extends AbstractScene implements IAnimatable, IClipPlanesHold
         // Use _activeCamera instead of activeCamera to avoid onActiveCameraChanged
         this._activeCamera = camera;
         this.setTransformMatrix(this._activeCamera.getViewMatrix(), this._activeCamera.getProjectionMatrix());
-        this.onAfterRenderCameraObservable.notifyObservers(camera);

```

Comment:
This one is a more problematic breaking change because the camera that is passed is not the camera passed to `_renderForCamera` in the loop above.

I think the code should be:
```javascript
if (!camera._useMultiviewToSingleView) {
    this.onAfterCameraRenderObservable.notifyObservers(camera);
}
```
The `if` is to avoid calling two times `onAfterCameraRenderObservable` in the `_useMultiviewToSingleView` case, as it seems it is what should be done. HOWEVER, it will break WebXR as the `onAfterRenderCameraObservable` observable is used by `WebXRExperienceHelper` (pinging @RaananW who will know better than me)...

So, for the time being, the existing code should be kept "as is" (        `this.onAfterRenderCameraObservable.notifyObservers(camera);`) to avoid breaking things.

Reply:
Rolled back these changes.